README.rst

```diff
@@ -109,7 +111,7 @@ https://git-scm.com/docs/git-submodule
 Installing the Images
 =====================
 
-See detailed instructions in doc/howtos/image-install.rst.
+See detailed instructions in ``doc/howtos/image-install.rst``.
```

Comment:
Alas, ReST does not support cross-file references (that's a Sphinx thing using :ref:`label name`).  ReST directly supports intra-page links and hyperlinks (http protocol), but while we're delivering the documentation on github like this, we can't reference the other document files.  I think this is a temporary situation until we have this project's documentation as part of the overall YP document set.